api/environments/migrations/0027_add_hide_disabled_flags.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 3.2.16 on 2023-01-05 03:01
+
+from django.db.models import Subquery, OuterRef
+from django.db import migrations, models
+
+
+def copy_hide_disabled_flags_config_from_project(apps, schema_editor):
```

Comment:
I think my comment above would negate the need for this too?

Reply:
Yep